src/test_handler.py

```diff
@@ -9,9 +10,9 @@ def test_handler(self):
         context = Mock()
         reader = Mock()
         mailer = Mock()
-        handler = lambda_handler(event=event, context=context, reader=reader, mailer=mailer)
+        dynamo = Mock()
+        current_date = datetime.today().strftime('%Y.%m.%d.%H.%M.%S')

```

Comment:
what if this command is executed at second X and the lambda call is executed at second X + 1?